streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -846,7 +853,7 @@ void runOnce() {
 
                     if (log.isDebugEnabled()) {
                         log.debug("Committed all active tasks {} and standby tasks {} in {}ms",
-                            taskManager.activeTaskIds(), taskManager.standbyTaskIds(), commitLatency);
+                            taskManager.activeRunningTaskIds(), taskManager.standbyTaskIds(), commitLatency);
```

Comment:
This is basically a renaming to make the code clearer. With the state updater restoring tasks are owned by the state updater.